tests/ops/api/v1/endpoints/test_system.py

```diff
@@ -0,0 +1,120 @@
+import json
+
+import pytest
+from fastapi_pagination import Params
+from sqlalchemy.orm import Session
+from starlette.testclient import TestClient
+
+from fides.api.ops.api.v1.scope_registry import (
+    CONNECTION_CREATE_OR_UPDATE,
+    CONNECTION_READ,
+    STORAGE_DELETE,
+)
+from fides.api.ops.api.v1.urn_registry import V1_URL_PREFIX
+from tests.ops.api.v1.endpoints.test_connection_config_endpoints import (
+    TestPatchConnections,
+)
+
+page_size = Params().size
+
+
+class TestPatchSystemConnections(TestPatchConnections):
+    @pytest.fixture(scope="function")
+    def url(self, system) -> str:
+        return V1_URL_PREFIX + f"/system/{system.fides_key}/connection"
+
+
+class TestGetConnections:
+    @pytest.fixture(scope="function")
+    def url(self, system) -> str:
+        return V1_URL_PREFIX + f"/system/{system.fides_key}/connection"
+
+    def test_get_connections_not_authenticated(
+        self, api_client: TestClient, generate_auth_header, connection_config, url
+    ) -> None:
+        resp = api_client.get(url, headers={})
+        assert resp.status_code == 401
+
+    def test_get_connections_wrong_scope(
+        self, api_client: TestClient, generate_auth_header, connection_config, url
+    ) -> None:
+        auth_header = generate_auth_header(scopes=[STORAGE_DELETE])
+        resp = api_client.get(url, headers=auth_header)
+        assert resp.status_code == 403
+
+    def test_get_connection_configs(
```

Comment:
Nice test - another test to add would be when the system id is invalid to test your `validate_system` code